packages/data-provider/src/utils/submittedDataUtils.ts

```diff
@@ -101,6 +95,23 @@ const utils = (dependencies: BaseDependencies) => {
 			return hasErrors;
 		},
 
+		/**
+		 * Parses an array of SubmittedData objects into a more compact form used as a respone
+		 * @param {SubmittedData[]} submittedData
+		 * @returns {SubmittedDataResponse[]}
+		 */
+		mapRecordsSubmittedDataResponse: (submittedData: SubmittedData[]): SubmittedDataResponse[] => {
+			return submittedData.map((data) => {
+				return {
+					data: data.data,
+					entityName: data.entityName,
+					isValid: data.isValid,
+					organization: data.organization,
+					systemId: data.systemId,
+				};
+			});
```

Comment:
Unless there's other manipulation meant to happen before the return, you could use the implicit one.
```suggestion
			return submittedData.map((data) => ({
				data: data.data,
				entityName: data.entityName,
				isValid: data.isValid,
				organization: data.organization,
				systemId: data.systemId,
			}));
```